install-script/install.sh

```diff
@@ -167,6 +178,17 @@ Supported env variables:
 EOF
 }
 
+# shellcheck disable=SC2317  # Don't warn about unreachable commands in this function
+# ShellCheck may incorrectly believe that code is unreachable if it's invoked in
+# a trap, like the reporter function.
+function reporter {
+    if ! $DISABLE_INSTALLATION_TELEMETRY; then
+        echo "SUMOLOGIC_INSTALLATION_TOKEN=${SUMOLOGIC_INSTALLATION_TOKEN}" >> "$INSTALLATION_LOGFILE"
+        curl --silent --location -X POST --data-binary @"${INSTALLATION_LOGFILE}" "${INSTALLATION_LOGFILE_ENDPOINT}"
```

Comment:
We should really delete the logfile here, if we put the installation token in it.